recipes/codecov/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "codecov" %}
+{% set version = "2.0.6" %}
+{% set sha256 = "999bf4d6f7e072433601798f3f291277a4ab519b245af2971cdeb247dbb21454" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+#  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/codecov/codecov-python/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - coverage
+    - requests >=2.7.9
+    - argparse  # [py26]
+  run:
+    - python
+    - coverage
+    - requests >=2.7.9
+    - argparse  # [py26]
```

Comment:
conda-forge doesn't build py26 packages so you don't need to include any packages that are py26 only. Delete argparse